builtin/files/userdata/cloud-config-controller

```diff
@@ -1074,6 +1074,11 @@ write_files:
         "${mfdir}/kube-dns-de.yaml"
       {{- end }}
       {{ if .KubeDns.NodeLocalResolver -}}
+      {{ if .KubeDns.dnsmasq.CoreDNSLocal.Enabled -}}
```

Comment:
Hey @kfr2, could we flip some of the `-` locations in this templating? We're trimming whitespace on the end here, causing start up to try and (in some cases) `deploy "${mfdir}/dnsmasq-node-coredns-local.yaml"deploy "${mfdir}/dnsmasq-node-ds.yaml"` which is causing issues.

Reply:
See [here](https://github.com/kubernetes-incubator/kube-aws/pull/1900)

Reply:
Sure thing!